lib/std/os.zig

```diff
@@ -4012,20 +4061,41 @@ pub fn realpathZ(pathname: [*:0]const u8, out_buffer: *[MAX_PATH_BYTES]u8) RealP
 
 /// Same as `realpath` except `pathname` is null-terminated and UTF16LE-encoded.
 /// TODO use ntdll for better semantics
```

Comment:
Looks like this TODO can be deleted now right?

Reply:
I wish it was so but not quite since `GetFinalPathNameByHandleW` is a kernel32 routine. I haven't figured out yet what to use in its place, but until I do (or other contributor), I'd like to leave the TODO in so that we don't forget about it. Now that I think of it, perhaps we could augment the comment to explain what's left here to do:

```suggestion
/// TODO use ntdll to emulate `GetFinalPathNameByHandleW` routine
```